test/e2e/testdata/TestIstioOperator/MeshGateway_sets_up_working_ingress.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: istio.banzaicloud.io/v1beta1
+kind: MeshGateway
+metadata:
+  labels:
+    app: mgw01
+    istio: mgw01
+  name: mgw01
+  namespace: istio-system
+spec:
+  labels:
+    app: mgw01
+    istio: mgw01
+  maxReplicas: 1
+  minReplicas: 1
+  replicaCount: 1
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 80
+  runAsRoot: true
```

Comment:
I wouldn't mind if we could test with this set to `false`, because that's the default and in that case there could be issues because of the less privileges.

Reply:
done